torch/onnx/_internal/fx/onnxfunction_dispatcher.py

```diff
@@ -533,7 +538,9 @@ def perfect_match_inputs(
     def _match_onnx_attribute_type(
         self,
         attribute_name: str,
-        attribute: fx_type_utils.Argument,
```

Comment:
Otherwise it crashes dispatcher with beartype error, when there is misalignment of input with attribute.
E.g., `aten::scalar_tensor` supports `self` as both sym number and python number. The former maps to onnx input and the latter maps to onnx attribute.